app/models/post_action_type.rb

```diff
@@ -100,6 +100,7 @@ def initialize_flag_settings
         notify_type: true,
         custom_type: true,
       )
+      @flag_settings.add(10, :illegal, topic_type: true, notify_type: true, custom_type: true)
```

Comment:
We have to skip `9` because of this hard-code over here:

https://github.com/discourse/discourse/blob/main/app/models/reviewable_score.rb#L14

Reply:
Perhaps drop a comment here, saying `ReviewableScore may add additional flag types` for the next poor soul :)